ci-operator/step-registry/ibmcloud/deprovision/vpc/ibmcloud-deprovision-vpc-commands.sh

```diff
@@ -101,6 +101,48 @@ echo "Using region: ${region}  resource_group: ${resource_group} vpc: ${vpc_name
 echo "DEBUG" "Removing the vpc ${vpc_name} ..."
 delete_vpc "${vpc_name}"
 
+hasError=0
+echo "DEBUG" "Checking the remains resources and the last_operation of the resources in ${resource_group}..."
+count=$(eval "${IBMCLOUD_CLI}" resource service-instances --type all -g ${resource_group} --output JSON | jq '.[]|.name' | wc -l)
+if [ "${count}" -gt 0 ]; then
+  echo "ERROR: has remain resource ..."
+  "${IBMCLOUD_CLI}" resource service-instances --type all -g ${resource_group} --output JSON | jq '.[]|.name+" "+.type+" "+.resource_id+" "+.crn'
```

Comment:
can we use multiple spaces or some other separators to clearly show the fields.

Reply:
how about use "|"? 
$  ic resource service-instances --type all --output JSON | jq '.[]|.name+"|"+.type+"|"+.resource_id+"|"+.crn'
"superglue-immovably-urn-cubicle|resource_instance|is.volume|crn:v1:bluemix:public:is:jp-tok-3:a/fdc2e14cf8bc4d53a67f972dc2e2c861::volume:r022-06da8a58-1d94-4735-a4c8-0e35eeb109e6"

Reply:
"|" -> "   |   "